routers/private/serv.go

```diff
@@ -61,6 +61,12 @@ func ServNoCommand(ctx *macaron.Context) {
 			})
 			return
 		}
+		if !user.IsActive {
+			ctx.JSON(http.StatusForbidden, map[string]interface{}{
+				"err": fmt.Sprintf("Your account is unactived."),
```

Comment:
```suggestion
				"err": fmt.Sprintf("Your account is disabled."),
```

Or maybe "inactive". Unactivated seems like a strange word.